BonusCalcListener/serverless.yml

```diff
@@ -24,6 +24,7 @@ functions:
       ENVIRONMENT:  ${ssm:/housing-tl/${self:provider.stage}/aspnetcore-environment}
       ### Reuse existing connection string from API (already definied in SSM)
       CONNECTION_STRING: Host=${ssm:/bonuscalc-api/${self:provider.stage}/postgres-hostname};Port=${ssm:/bonuscalc-api/${self:provider.stage}/postgres-port};Database=${ssm:/bonuscalc-api/${self:provider.stage}/postgres-database};Username=${ssm:/bonuscalc-api/${self:provider.stage}/postgres-username};Password=${ssm:/bonuscalc-api/${self:provider.stage}/postgres-password}
+      TZ: Europe/London

```

Comment:
Need to run the app in the Europe/London timezone so that the date calculations work out correctly.